packages/accounts-base/accounts_server.js

```diff
@@ -1135,13 +1135,15 @@ Ap._expirePasswordResetTokens = function (oldestValidDate, userId) {
   var userFilter = userId ? {_id: userId} : {};
 
   this.users.update(_.extend(userFilter, {
+    "services.password.reset.reason": reason,
     $or: [
       { "services.password.reset.when": { $lt: oldestValidDate } },
       { "services.password.reset.when": { $lt: +oldestValidDate } }
     ]
   }), {
     $unset: {
       "services.password.reset": {
```

Comment:
So, I think this is completely bogus. `$unset` takes a field name and does not care about the value of the filedname. That $or there is whatever?

This could be simply simplified to:

```
$unset: {
  "services.password.reset": ""
}
```